packages/react-inbox/src/hooks/use-click-outside.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from "react";
+
+export default (ref, handler) => {
```

Comment:
could we use named export? if not, I would still suggest adding named functions, and then default export that. It helps with future debugging.